src/new-client/src/app/site/[site]/view-availability/week/page.tsx

```diff
@@ -0,0 +1,22 @@
+import NhsPage from '@components/nhs-page';
+import dayjs from 'dayjs';
+
+type PageProps = {
+  searchParams: {
+    from: string;
+    until: string;
+  };
+};
+
+const Page = async ({ searchParams }: PageProps) => {
```

Comment:
I think we might need to add `await assertPermission(site.id, 'availability:query');` here, same as its parent page?

Reply:
One day I will remember to add that in to a new page :D

Reply:
I did consider making it a required parameter of `<NhsPage/>` and `<NhsTransactionalPage/>` for just that reason :D Feel free to do so, I only held off cos I wasn't sure if there'd be pages with an AND/OR condition. Pretty sure atm though it's just one role or none so would be trivial